tests/test_ble_interface_core.py

```
@@ -4947,7 +5087,113 @@ def _record_noop_start(thread: object) -> None:
+        monkeypatch.setattr(
+            iface.thread_coordinator,
+            "_create_thread",
+            lambda **_kwargs: created_threads.pop(0),
+            raising=True,
+        )
+
+        start_calls: list[object] = []
+        monkeypatch.setattr(
+            iface.thread_coordinator,
+            "_start_thread",
+            lambda thread: start_calls.append(thread),
```

Ruff (0.15.6)

[warning] Lambda may be unnecessary; consider inlining inner function

Inline function call

(PLW0108)